go/kbfs/libkbfs/disk_block_cache.go

```diff
@@ -897,7 +897,7 @@ func (*DiskBlockCacheLocal) getRandomBlockID(numElements,
 	}
 	// Generate a random block ID to start the range.
 	pivot := (1.0 - (float64(numElements) / float64(totalElements)))
-	return kbfsblock.MakeRandomIDInRange(0, pivot)
+	return kbfsblock.MakeRandomIDInRange(0, pivot, fakeRandForTest)
```

Comment:
You can avoid passing in the bool above by using `cache.config.IsTestMode()` here instead, and naming the receiver of the function `cache`.